benchmarks/portable/awa-python-bench/main.py

```
@@ -242,6 +262,240 @@ async def handle(_job: awa.Job[ChaosJob]) -> None:
+def _now_iso() -> str:
+    import datetime as _dt
+
+    return (
+        _dt.datetime.now(_dt.timezone.utc)
+        .isoformat(timespec="milliseconds")
+        .replace("+00:00", "Z")
+    )
+
+
+def _percentiles(
+    events: "collections.deque[tuple[float, float]]",
```

Ruff (0.15.10)

[error] Undefined name `collections`

(F821)